go/vt/vttablet/tabletmanager/vreplication/vcopier_test.go

```diff
@@ -246,6 +246,267 @@ func TestPlayerCopyBigTable(t *testing.T) {
 	})
 }
 
+// TestPlayerCopyWildcardRule ensures the copy-catchup back-and-forth loop works correctly
+// when the filter uses a wildcard rule
+func TestPlayerCopyWildcardRule(t *testing.T) {
```

Comment:
This doesn't seem to be part of the PR. Was this a test we discussed adding?

Reply:
I had the lastpk change on this branch because that PR had not been merged yet. Will rebase.

Reply:
done.